lib/engine/game/g_18_christmas_eve/game.rb

```diff
@@ -294,6 +301,16 @@ def operating_round(round_num)
           ], round_num: round_num)
         end
 
+        def status_array(corporation)
+          return [] unless @round.respond_to?(:presidencies_gifted)
+
+          if @round.presidencies_gifted.include? corporation
```

Comment:
missing parens

Reply:
As in, `if (@round.presidencies_gifted.include? corporation)` or `@round.presidencies_gifted.include?(corporation)`?

Reply:
latter, x.y?()

Reply:
Resolved